spec/features/operator/pipeline/install_test.go

```diff
@@ -0,0 +1,14 @@
+package pipeline
+
+import (
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestPipelinesInstall(t *testing.T) {
+	Convey("Given the Operator is installed", t, func() {
+		Convey("It should install pipelines controllers as a deployment to target namespace (openshift-pipelines)", nil)
```

Comment:
We could rephrase this into `it should have installed …` right ? as we will "only" verify that the thing are present, aka installed.

Reply:
right it should be``` it should have installed```